src/pimp_mixer_arm.S

```diff
@@ -20,15 +20,14 @@
 @irq-safe:   1121327
 @irq-unsafe: 1105083
 
-// #define IRQ_SAFE
+#define IRQ_SAFE /* on by default */
 #define USE_BRESENHAM_MIXER
 // #define NO_MIXING
 // #define NO_CLIPPING
 
+#ifndef IRQ_SAFE
 .stack_store:
 .word 0
-
-#ifndef IRQ_SAFE
```

Comment:
So, this hunk makes sure that stack_store doesn't take up space when it's not used? Nice, but could perhaps be a separate commit for reviewability...

Reply:
Fair enough.